mcp/hook-fastmcp.py

```
@@ -0,0 +1,12 @@
+from PyInstaller.utils.hooks import collect_data_files
+
+# Collect all data files from fastmcp package
+datas = collect_data_files('fastmcp')
+
+# Try to collect metadata manually
+try:
+    import pkg_resources
+    dist = pkg_resources.get_distribution('fastmcp')
+    datas += [(dist.egg_info, 'fastmcp-' + dist.version + '.dist-info')]
+except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)